spec/postrank-uri_spec.rb

```diff
@@ -194,14 +194,21 @@ def c(uri)
       end
     end
 
-    it "preserves encoded question marks in the path" do
-      c('http://en.wikipedia.org/wiki/Whose_Line_Is_It_Anyway%3F_%28U.S._TV_series%29').
-        should == 'http://en.wikipedia.org/wiki/Whose_Line_Is_It_Anyway%3F_(U.S._TV_series)'
-    end
+    context "reserved characters" do
```

Comment:
Could we introduce the `context` in the previous commit to make this diff tidier?